releases/v18_1_0.rst

```diff
@@ -0,0 +1,29 @@
+#########################
+LSST Stack 18.1.0 Release
+#########################
+
+Version 18.1.0 of the LSST Science Pipelines is now available.
+It is based on 18.0.0 release (2019 weekly 23).
+
+LSST Science Pipelines
+======================
+
+Full details of the LSST Science Pipelines 18.1.0 release can be found in the :ref:`extensive release notes <release-v18-1-0>`.
+
+No characterization report is provided with this release.
+
+Qserv and Data Access
```

Comment:
I propose that we drop the Qserv and SUI entries from this page, as they do not add anything.